NeoMathEngine/include/NeoMathEngine/SimdMathEngine.h

```diff
@@ -32,6 +39,8 @@ class ISimdMathEngine : public CCrtAllocatedObject {
 
 	virtual void BlobConvolution( const CConvolutionDesc& convDesc, const float* source,
 		const float* filter, const float* freeTerm, float* result ) const = 0;
+
+	virtual SgemmFunc GetSgemmFunction() const = 0;
```

Comment:
Why Get...? Why not virtual void Sgemm(...) const = 0 ?

Reply:
_**virtual void Sgemm**_ doesn't bring clarity in calling code.